### PR TITLE
Updated the new csat 2025

### PR DESCRIPTION
**Pull request by rajeev456**

### CODE REVIEW

File: src/components/prelims-kaushal/PrelimsKaushal.js
Comment (jaichaudhary) [inline src/components/prelims-kaushal/PrelimsKaushal.js:1]: Change the text

File: src/components/prelims-kaushal/PrelimsKaushal.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import './PrelimsKaushal.css'
+
+const PrelimsKaushal = () => {
+  return (
+    <div className='heading'>This page us not available right now!</div>
```
Comment (jaichaudhary) [inline src/components/prelims-kaushal/PrelimsKaushal.js]: Typing mistake, correct it

File: package-lock.json
Comment (jaichaudhary) [inline package-lock.json:1]: Delete this file

File: src/components/Csat2025/Csat2025.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+      <h1 className="csat-title">
+        CSAT COURSE 2025
+      </h1>
+      <div className="highlight-line"></div>
+      <button className="batch-button">Batch 1</button>
+
+      <div className="content-section">
+        <div className="info-section">
+          <p className="csat-description">
+            The best course for CSAT 2025 with comprehensive coverage of all tropics, with through understanding supported by practice test, handouts, PYQ analysis, test series and mentorship.
```
Comment (jaichaudhary) [inline src/components/Csat2025/Csat2025.js]: Typing mistake, correct it

Comment (rajeev456) [reply src/components/Csat2025/Csat2025.js]: Done

**Review by jaichaudhary — APPROVED** [review]

**Pull request merged**